pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -192,12 +195,11 @@ func (r *ReconcileBareMetalHost) Reconcile(request reconcile.Request) (result re
 	annotations := host.GetAnnotations()
 	if annotations != nil {
 		if _, ok := annotations[metal3v1alpha1.PausedAnnotation]; ok {
-			return reconcile.Result{Requeue: true, RequeueAfter: pauseRetryDelay}, nil
```

Comment:
Can delete the definition of `pauseRetryDelay` also now I think.

Reply:
done